### ISSUE TITLE
build options documentation clarification desired

### ISSUE THREAD

**Issue by raould** [urho3d/Urho3D#698]
http://urho3d.github.io/documentation/1.32/_building.html#Build_Options

I am guessing that the middle column that has the title "V" is the default value for the flag in the first column of that row? Overall I find it unclear. And confusing because then sometimes the default value is explained in the description in the 3rd column. 

If the middle column *is* supposed to be the default, I will write up a p.r. to show what I hope might be a slightly more clear setup for the table in the docs.

**Comment by raould** [comment]
Correction: I guess what is really desired is "1) cd localrepo/ 2) mkdir build 3) ./cmake_generic.sh build 4) cd build 5) make"?

**Comment by weitjong** [comment]
The "V" stands for "Value". Our website in general and our online documentation specifically have a space constraint when viewed on mobile browser, e.g. iPhone Safari. So far it has these possible values: ```0```, ```1```, ```-```, ```*```
I suppose the first three are self-explanatory (but I could be wrong judging from your issue). It means the option is being set with the given value, 0, 1, or uninitialized.
The ```*``` is the confusing part. When we first use it, the following text in the next column would supplement what the ```*``` value actually means in its own context. But in the latter documentation update, it seems we have not followed this strictly. So, yeah, I think that needs to be fixed.

**Comment by weitjong** [comment]
A minor correction on my earlier comment regarding the the space constraint on small viewport browsers. Although the space constraint still largely exists, I forgot that we have already improved our website designed which use Bootstrap's ```responsible table``` so for this particular case we can actually use the full text if necessary. And I have just made that change to the master branch.

You are still welcome to send your PR to make further changes. Thus I leave this issue as open for now.
